cv32e40x_manifest.flist

```diff
@@ -74,4 +74,7 @@ ${DESIGN_RTL_DIR}/cv32e40x_pc_target.sv
 
 ${DESIGN_RTL_DIR}/../bhv/cv32e40x_sim_clock_gate.sv
 ${DESIGN_RTL_DIR}/../bhv/cv32e40x_rvfi.sv
+${DESIGN_RTL_DIR}/../bhv/cv32e40x_rvfi_instr_obi.sv
+${DESIGN_RTL_DIR}/../bhv/cv32e40x_rvfi_data_obi.sv
+${DESIGN_RTL_DIR}/../bhv/cv32e40x_rvfi.sv
```

Comment:
cv32e40x_rvfi.sv mentioned twice in manifest.

Reply:
Fixed now